website/src/pages/download.md

```diff
@@ -14,33 +14,6 @@ https://www.apache.org/dyn/closer.lua/incubator/opendal/0.44.1/apache-opendal-in
 
 For older releases, please check the [archive](https://archive.apache.org/dist/incubator/opendal/).
 
-## Notes
+## Verifying Hashes and Signatures
 
-* When downloading a release, please check the SHA-512 and verify the OpenPGP compatible signature from the main Apache site. Links are provided above (next to the release download link).
```

Comment:
I think what needs to be changed the most is just this sentence.😯

Reply:
Addressed.